README.md

```diff
@@ -64,7 +70,14 @@ import { getConfig } from '@imtbl/core-sdk';
 const ethNetwork = 'ropsten'; // or mainnet;
 
 // Use the helper function to get the config
-const config = getConfig(ethNetwork);
+const config = getConfig({
+  starkContractAddress: '0x4527BE8f31E2ebFbEF4fCADDb5a17447B27d2aef',
+  registrationContractAddress: '0x6C21EC8DE44AE44D0992ec3e2d9f1aBb6207D864',
+  chainID: 3,
+  apiConfigOptions: {
+    basePath:  'https://api.ropsten.x.immutable.com',
```

Comment:
This Ropsten stuff is literally about to change - what's the timing @shineli1984 ?